README.md

```diff
@@ -24,6 +25,48 @@ An operator can dynamically update a Node's role by promoting a Worker to Manage
 
 *Tasks* are organized in *Services*. A service is a higher level abstraction that allows the user to declare the desired state of a group of tasks.  Services define what type of task should be created as well as how to execute them (e.g. run this many instances at all times) and how to update them (e.g. rolling updates).
 
+## Features
+
+Some of *SwarmKit*'s main features are:
+
+- **Orchestration**
+  - **Desired State Reconciliation**: *SwarmKit* will constantly compare the desired state against the current cluster state
+    and reconcile the two if necessary. For instance, if a node fails, *SwarmKit* will reschedule its tasks onto a different node.
+  - **Service Types**: There are different types of services. The project currently ships with two of them out of the box:
+    - **Replicated Services** can be scaled to a given number of replicas.
+    - **Global Services** will run one task on every node in the cluster.
+  - **Configurable Updates**: Every field of a service can be changed at any time.
```

Comment:
At any time, you can change the value of one or more fields for a service. After you make the update, SwarmKit reconciles the desired state...